FutureNorthants/northants-website
# Do not archive pages past review date

### Trello card link

https://trello.com/c/T0xHkETP/277-p1-page-review-dates-report-bug

### Setup
1. You'll need to modify nc_nodes_form_node_validate_date_time() in the nc_nodes.module file to permit setting a review date in the past. You can do this by changing line 113 so that instead of reading `if ($date > $max_date || $date < $today) {` it reads `if ($date > $max_date) {`.

### Testing
1. Set a published page's review date to be in the past (more than 30 days ago in particular) and save it in published state.
2. Verify the page shows up in the Past Review tab of the Content dashboard.
3. Run the Drupal cron (Configuration | System | Cron | Run Cron)
4. Verify the page is not in the archived state afterwards (easiest shown on the Reviews tab of the page).
5. Verify the page still shows up in the Past Review tab.
